lib/config/go_routes/go_router.dart

```diff
@@ -357,13 +357,20 @@ abstract class AppRoutes {
             GoRoute(
               path: ':roomid',
               pageBuilder: (context, state) {
-                final shareFile = state.extra as MatrixFile?;
+                if (state.extra == 'fromDraftChat') {
+                  return NoTransitionPage(
+                    child: Chat(
+                      roomId: state.pathParameters['roomid']!,
+                      key: Key(state.pathParameters['roomid']!),
+                    ),
+                  );
+                }
```

Comment:
WHEN USE BACK ON WEB BROWSER WHAT IS HAPPEN ?

Reply:
VERRRYYY BADD  `'fromDraftChat'`.

Reply:
check the demo